scripts/framework-health.sh

```
@@ -0,0 +1,285 @@
+                *)
+                    echo -e "  ⚪ $role: ${YELLOW}$STATUS${NC} (updated: $LAST_UPDATED)"
+                    ;;
+            esac
+        fi
+    done
+    
+    echo ""
+    echo -e "${YELLOW}Active Work Distribution:${NC}"
+    ACTIVE_AGENTS=$(grep -l "ACTIVE" docs/development/roles/*.md 2>/dev/null | wc -l | tr -d ' ')
+    AVAILABLE_AGENTS=$(grep -l "Available" docs/development/roles/*.md 2>/dev/null | wc -l | tr -d ' ')
+    TOTAL_AGENTS=$(ls docs/development/roles/*.md | grep -v ROLE-TEMPLATE | grep -v README | wc -l | tr -d ' ')
```

Shellcheck (0.10.0)

[warning] Don't use ls | grep. Use a glob or a for loop with a condition to allow non-alphanumeric filenames.

(SC2010)